operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java

```diff
@@ -401,8 +437,8 @@ private String validatePath(String s) {
    * @since 2.0
    * @return log home enabled
    */
-  boolean isLogHomeEnabled() {
-    return Optional.ofNullable(logHomeEnabled).orElse(!isDomainHomeInImage());
```

Comment:
In the old code, if logHomeEnabled is not set, and the domain is on PV, the logHomeEnabled will be set to true. This is missing in the new code. Is this intentional.  The doc seems to indicate that the old logic remains; so the doc and the code does not match.

Reply:
@jshum2479 Please follow up.  Make sure LogHomeEnabled defaults to true for PV, and false for FromModel and Image.

Reply:
The logic is in Domain.java isLogHomeEnabled